table/table_test.go

```diff
@@ -45,11 +45,11 @@ var defaultBuilderOpt = options.TableBuilderOptions{
 		BitsPerKeyHint: 40,
 		RealSuffixLen:  10,
 	},
-	Compression:         options.ZSTD,
+	CompressionPerLevel: []options.CompressionType{options.ZSTD},
 	BlockSize:           4 * 1024,
 	HashUtilRatio:       0.75,
 	WriteBufferSize:     1024 * 1024,
-	MaxLevels:           7,
+	MaxLevels:           1,
```

Comment:
Why change this?

Reply:
The table size is small, we can still write multiple level tables.

Reply:
Tests in `table` package only test on one tables, and has no concept about level.

Reply:
Tests in table package only test on one table, and has no concept about level.